frontend-zs/src/components/PrivateRoute.js

```diff
@@ -3,11 +3,13 @@ import { Route, Redirect } from 'react-router-dom';
 
 import { authenticationService } from '../services/authentication.service';
 
-export const PrivateRoute = ({ component: Component }) => (
-  <Route render={props => {
-    const currentUser = authenticationService.currentUserValue;
-    if (!currentUser) return <Redirect to={{ pathname: '/login' }} />
-    if (!currentUser.registrationComplete) return <Redirect to={{ pathname: '/registrationStepTwo' }} />
-    return <Component/>
-  }} />
+export const PrivateRoute = ({ component: Component, ...rest }) => (
+  <Route
+    {...rest}
+    render={props => {
+      const currentUser = authenticationService.currentUserValue;
+      if (!currentUser) return <Redirect to={{ pathname: '/login' }} />
+      if (!currentUser.registrationComplete) return <Redirect to={{ pathname: '/registrationStepTwo' }} />
+      return <Component />
+    }} />
```

Comment:
```suggestion
export const PrivateRoute = ({ component: Component, ...rest }) => (
  <Route
    render={props => {
      const currentUser = authenticationService.currentUserValue;
      if (!currentUser) return <Redirect to={{ pathname: '/login' }} />
      if (!currentUser.registrationComplete) return <Redirect to={{ pathname: '/registrationStepTwo' }} />
      return <Component {...props} {...rest}/>
    }} />
```
In addition to changing `StoryPage()` to `StoryPage(props)`, will lead to the availability of the params in `props.computedMatch.params`. I think it's better to send all the props to the components as it would contain the history and some other useful information (might be needed by other components using PrivateRoute).  What do you think?